.circleci/config.yml

```diff
@@ -427,21 +427,44 @@ jobs:
     environment:
       TERM: xterm
       CMAKE_BUILD_TYPE: Debug
-      CMAKE_OPTIONS: -DLLL=ON
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - dependencies-osx-{{ .Branch }}-{{ checksum ".circleci/config.yml" }}
```

Comment:
Actually this should also include the checksum of ``.circleci/osx_install_dependencies.sh``, but I'd say we can still merge and change that in the Z3 4.8.6 follow-up PR that'll happen soon anyways.